pkg/asset/manifests/gcp/loadbalanceruid.go

```diff
@@ -0,0 +1,45 @@
+package gcp
+
+import (
+	"crypto/md5"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// UIDConfigMapName is the Key used to persist UIDs to configmaps.
+	UIDConfigMapName = "ingress-uid"
+
+	// UIDNamespace is the namespace which contains the above config map
+	UIDNamespace = metav1.NamespaceSystem
+
+	// UIDCluster is the data keys for looking up the clusters UID
+	UIDCluster = "uid"
+
+	// UIDProvider is the data keys for looking up the providers UID
+	UIDProvider = "provider-uid"
+)
+
+func GenerateLoadBalancerUID(infraID string) string {
```

Comment:
golint error: switch this to private -> `generateLoadBalancerUID`

Reply:
I wanted a public function that I can call from the destroy code so it knows how to generate the LB UID as well. Maybe this is not the best place for it. I will move it.

Reply:
oh that's fine. then just add a comment. sorry, I did a cursory glance and saw it wasn't used anywhere else so assumed it was private.

Reply:
(golint is failing because the exported function doesn't have a comment).

Reply:
@patrickdillon I moved it to the types package because I don't want to introduce a dependency from the destroy pkg to assets. Is that ok?

Reply:
Seems good to me!